scripts/setup.sh

```diff
@@ -318,6 +335,9 @@ configure_docker_nvidia
 # Install Rust
 install_rust
 
+# Install Just
+install_rust
```

Comment:
```suggestion
install_just
```

Reply:
god damn it, thank you @austinabell  🤦‍♂️